history-of-the-quakers/original/31-book-3-chapter-5.adoc

```diff
@@ -142,11 +152,18 @@ in which we have peace: and that we may not be brought into ungodliness by them.
 So hear, and consider, and do good in thy time, whilst thou hast power; and be merciful,
 and forgive; that is the way to overcome, and obtain the kingdom of Christ.
 
-GEORGE FOX.`"
+[.signed-section-signature]
+George Fox
+
+--
 
+[.offset]
```

Comment:
hmmm... i think it would be slightly better to move this into the embedded section below, and make it a `[.letter-heading]` -- it's a little fuzzy, but I think that's what I would do in this situation.